postgres/parser/parser/sql/sql_parser.go

```diff
@@ -82,3 +83,7 @@ func (p *PostgresParser) ParseOneWithOptions(_ context.Context, query string, _
 	}
 	return vitessAST, 0, nil
 }
+
+func (p *PostgresParser) QuoteIdentifier(identifier string) string {
+	return fmt.Sprintf("\"%s\"", strings.ReplaceAll(identifier, "\"", "\"\""))
```

Comment:
Seems like it would be better to use backticks here for clarity.

Reply:
good catch